datasync/chaindatafetcher/chaindata_fetcher.go

```diff
@@ -1,6 +1,7 @@
 package chaindatafetcher
```

Comment:
Please add a license description.

Reply:
I will add the license of this file on another PR.